src/upload-tests.proj

```diff
@@ -63,8 +63,8 @@
     <!-- Properties used for submission by CloudTest.Helix.Targets-->
     <BuildMoniker>$(CurrentDate)</BuildMoniker>
     <BuildMoniker Condition="'$(IsOfficial)'=='true'">$(OfficialBuildId)</BuildMoniker>
-    <!-- TODO:  This requires some specific understanding of Helix, we should try to generalize this into properties.  -->
-    <HelixJobProperties>{ &quot;architecture&quot; : &quot;$(ArchGroup)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetOS)&quot; }</HelixJobProperties>
+    <HelixArchLabel>$(ArchGroup)</HelixArchLabel>
```

Comment:
While this is a step in the right direction. I think it would still be better if we directly pass the HelixArchLabel and HelixConfigLabel directly from the build definition so we aren't dependent on the common targets which are catering to individual project builds to set these correctly for this helix run.